data/json/monsters/mammal.json

```diff
@@ -1508,7 +1524,7 @@
   {
     "id": "mon_dog_gshepherd_pup",
     "type": "MONSTER",
-    "name": { "str": "German Shepherd puppy", "str_pl": "German Shepherd puppies" },
+    "name": { "str": "german Shepherd puppy", "str_pl": "german Shepherd puppies" },
```

Comment:
```suggestion
    "name": { "str": "german shepherd puppy", "str_pl": "german shepherd puppies" },
```

Reply:
German Shepard is an official breed, both words are supposed to be capitalized.